### PR TITLE
Change package name and add YPlan maintainer.

### PR DESCRIPTION
**Pull request by seporaitis**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -27,13 +27,15 @@
     license = f.read()
 
 setup(
-    name='salesforce-oauth-request',
+    name='salesforce-oauth-request-yplan',
     version='1.1.0',
```
Comment (adamchainz) [inline setup.py:31]: maybe bump version too? or at least when you release

Comment (seporaitis) [inline setup.py:31]: It's bumped up - upstream is at `1.0.6`.

Comment (seporaitis) [inline setup.py:31]: Bumped up as part of #1

**Comment by seporaitis** [comment]
Will merge then and publish!

**Pull request merged**